main.go

```diff
@@ -32,6 +41,11 @@ func main() {
 			inputAsInt[i][sid] = value
 		}
 
+		if *fieldsParameter != "" && *charParameter != "" {
```

Comment:
Please explain why `==` would be better than `&&`.

Reply:
i dont understand the question i cannot use `==`?

Reply:
`==` also covers neither -f, nor -c being given which you don't want.

Reply:
still dont understand it

Reply:
Shall your program be able to run without -f and -c?

Reply:
no it cannot be used without parameters

Reply:
If you run just `go run .`, both `*fieldsParameter != ""` `*charParameter != ""` are false – and equal. The same for `go run . -f 1 -c 2`, but true – also equal. With `==` you cover both invalid cases.